python/cudf/cudf/testing/io.py

```diff
@@ -0,0 +1,78 @@
+import copy
+import json
+import logging
+import os
+import random
+import sys
+
+logging.basicConfig(
+    format="%(asctime)s %(levelname)-8s %(message)s",
+    level=logging.INFO,
+    datefmt="%Y-%m-%d %H:%M:%S",
+)
+
+
+class IOBase(object):
```

Comment:
Can we rename this class? `IOBase` is a class in the Python `io` standard library that we don't want to mix up.